src/components/pages/Theme/Theme.jsx

```diff
@@ -428,6 +429,87 @@ export const Theme = () => {
         </Segment>
       </div>
 
+      <div className='section'>
+        <Segment padded>
+          <Header as='h2' dividing>
+            Table
+          </Header>
+          <Table>
+            <Table.Header>
+              <Table.Row>
+                <Table.HeaderCell>Transaction Name</Table.HeaderCell>
+                <Table.HeaderCell>Status</Table.HeaderCell>
+                <Table.HeaderCell textAlign='right'>Amount</Table.HeaderCell>
```

Comment:
The `textAlign` prop is helpful but let's only apply it on bigger screens. I think everything should be left aligned when the table is stacked vertically.

![Screenshot 2023-06-16 at 10 08 37 AM](https://github.com/abby-moffitt/GA-Semantic/assets/1568524/842e5559-0e0d-43e6-8240-2c7cbc7ebb77)